src/tests/test_dynamic_api/test_temporal_actions.py

```diff
@@ -180,3 +180,16 @@ def test_serializer_temporal_request_corrects_link_to_temporal(
         assert data["_links"]["bestaatUitBuurten"][0]["href"].endswith(expected_url), data[
             "bestaatUitBuurten"
         ]
+
+    def test_correct_handling_of_extra_url_params(
+        self, api_client, reloadrouter, stadsdelen, buurt
+    ):
+        """Prove that extra url parameters do not interfere with the existing
+        url parameters for temporality."""
+        url = reverse("dynamic_api:gebieden-stadsdelen-list")
+        response = api_client.get(f"{url}?_format=json")
+        data = read_response_json(response)
+
+        # Check if there is only one '?' in the temporal urls
+        assert len(data["_embedded"]["stadsdelen"][1]["_links"]["self"]["href"].split("?")) == 2
+        assert len(data["_embedded"]["stadsdelen"][1]["_links"]["wijk"][0]["href"].split("?")) == 2
```

Comment:
waarom niet de parameters zelf checken? :-)   `"?_format=json&..." in ...` bijvoorbeeld.

Reply:
Ja, wellicht nog wat duidelijker. Alhoewel, je er niet van uit kunt gaan dat de volgorde van de parameters vastligt, maar daar kun je rekening mee houden.

Reply:
Ik doet wel iets obv. urlparse, dat is nog wat explicieter.